modules/tools/src/main/scala/de/proteinevolution/tools/ToolConfig.scala

```diff
@@ -37,47 +33,59 @@ class ToolConfig @Inject() (config: Configuration, paramAccess: ParamAccess) {
 
   lazy val version: String = config.get[String]("version")
 
-  lazy val values: Map[String, Tool] = {
-    config.get[Config]("Tools").root.asScala.map {
-      case (_, configObject: ConfigObject) =>
-        val config    = configObject.toConfig
-        val inputType = Try(config.getString("input_type")).getOrElse(TextAreaInputType.SEQUENCE)
-        config.getString("name") -> toTool(
-          config.getString("name"),
-          config.getString("longname"),
-          config.getInt("order"),
-          config.getString("description"),
-          config.getString("code"),
-          config.getString("section").toLowerCase,
-          config.getString("version"),
-          config
-            .getStringList("parameter")
-            .asScala
-            .map { param =>
-              paramAccess.getParam(
-                param,
-                config.getString("placeholder_key"),
-                config.getString("sample_input_key"),
-                inputType
-              )
-            }
-            .toSeq,
-          // TODO remove Try when implemented for each tool
-          Try(
+  implicit val ec: scala.concurrent.ExecutionContext = scala.concurrent.ExecutionContext.global
+
+  private val watcher = new FileMonitor("tel/paramspec".toFile, recursive = true) {
+    override def onModify(file: File, count: Int): Unit = {
+      println(f"[TOOLS_CONFIG] refresh; loading new file contents for ${file.path.toString}")
```

Comment:
you could use the logger from the `Logging`  trait by using `import play.api.Logging`